packages/workflow/src/Extensions/ArrayExtensions.ts

```diff
@@ -344,12 +305,14 @@ function intersection(value: unknown[], extraArgs: unknown[][]): unknown[] {
 	}
 	const newArr: unknown[] = [];
 	for (const v of value) {
-		if (others.findIndex((w) => deepCompare(w, v)) !== -1) {
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-call
```

Comment:
Why do we need these `@typescript-eslint/no-unsafe-call` lines? It seems like there are no warnings/errors without them.

Reply:
Looks like we don't. Removing.